app/settings.py

```diff
@@ -38,7 +38,8 @@ def get_env_or_fail(key):
     return value
 
 
-CDN_URL = os.getenv("CDN_URL", "https://cdn.ons.gov.uk/sdc")
+CDN_URL = os.getenv("CDN_URL", "https://cdn.ons.gov.uk")
+CDN_ASSETS_PATH = os.getenv("CDN_ASSETS_PATH", "/sdc/design-system/")
```

Comment:
If we make the change I suggested in the deign system pr, this should be:
```suggestion
CDN_ASSETS_PATH = os.getenv("CDN_ASSETS_PATH", "/sdc/design-system")
```